.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 .sbtopts
 project/.sbt
+.idea
```

Comment:
Please remove this as it is developer's preference. You should have it in your git globals.